code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -317,6 +329,22 @@
 	level_min = 8
 	level_max = 8
 
+/datum/chemical_reaction/mix_virus/mix_virus_13
+
```

Comment:
Get rid of these empty lines

Reply:
That's how all the previous mix_viruses are formatted.